### DISCUSSION TITLE
Issue with displaying the <v-treeview> component in Vue.j,s , the display page is empty.

### DISCUSSION THREAD

**Discussion by chanezAMK** [vuejs/router#2185] [Help and Questions] [answered]
**Package.json**
`"name": "project.front.client",
  "version": "0.0.0",
  "private": true,
  "type": "module",
  "scripts": {
    "dev": "vite",
    "build": "run-p type-check \"build-only {@}\" --",
    "preview": "vite preview",
    "build-only": "vite build",
    "type-check": "vue-tsc --build --force",
    "lint": "eslint . --ext .vue,.js,.jsx,.cjs,.mjs,.ts,.tsx,.cts,.mts --fix --ignore-path .gitignore"
  },
  "dependencies": {
    "bootstrap": "^5.3.3",
    "vue": "^3.4.21",
    "vue-router": "^4.3.0",
    "vuetify": "^3.5.11"
  },
  "devDependencies": {
    "@rushstack/eslint-patch": "^1.3.3",
    "@tsconfig/node20": "^20.1.2",
    "@types/bootstrap": "^5.2.10",
    "@types/node": "^20.11.25",
    "@types/vue-router": "^2.0.0",
    "@vitejs/plugin-vue": "^5.0.4",
    "@vue/eslint-config-typescript": "^12.0.0",
    "@vue/tsconfig": "^0.5.1",
    "eslint": "^8.49.0",
    "eslint-plugin-vue": "^9.17.0",
    "npm-run-all2": "^6.1.2",
    "typescript": "~5.4.0",
    "vite": "^5.1.6",
    "vue-tsc": "^2.0.6"
  }`
**App.vue**
![image](https://github.com/vuejs/router/assets/157361842/24f36656-48f9-44bf-886b-1aec02572f12)
**FileExplorer.vue**
![image](https://github.com/vuejs/router/assets/157361842/a255ffe7-11a0-4550-ba6e-c8caa73e0632)
**error**
![image](https://github.com/vuejs/router/assets/157361842/e6693432-0cb9-4285-be0e-328b623bbb0a)

**Comment by skirtles-code** [comment, marked as answer]
This doesn't appear to be a question about Vue Router. This Discussions section isn't the right place to ask questions about Vuetify, or general Vue questions.

My best guess is that you need to follow the instructions at https://vuetifyjs.com/en/components/treeview/#installation to install that component. But someone may know more if you ask on one of the Vuetify support platforms, e.g. the Vuetify Discord server.
